www/include/monitoring/status/Services/xml/makeXMLForOneHost.php

```diff
@@ -83,6 +83,16 @@
 $disable = $obj->checkArgument("disable", $_GET, "disable");
 $dateFormat = $obj->checkArgument("date_time_format_status", $_GET, "Y/m/d H:i:s");
 
+$host_id = filter_var(
+    $host_id ?? null,
+    FILTER_SANITIZE_NUMBER_INT
+);
+
+if (!$host_id) {
```

Comment:
shouldn't we check ACL rights too ?
eg : `http://10.30.2.202/centreon/include/monitoring/status/Services/xml/makeXMLForOneHost.php?&host_id=35` changing the id of the host, make data leak possible